src/sbvr-api/permissions.coffee

```diff
@@ -209,8 +209,9 @@ exports.setup = (app, sbvrUtils) ->
 			resource: 'permission'
 			passthrough: req: rootRead
 			options:
-				select: 'name'
-				filter: permsFilter
+				$select: 'name'
+				$filter: permsFilter
+				$orderby: name: 'asc'
```

Comment:
Why are we ordering here?

Reply:
Yeah it's to increase hit rate for memoisation whilst keeping the cost of ordering minimal - I'll add a comment

Reply:
done